Makefile

```diff
@@ -0,0 +1,5 @@
+all:
+	 docker build -t postgresql-ulid --build-arg="PG_MAJOR=16" .
+	 
+run:
+	docker run -p 5432:5432 postgresql-ulid
```

Comment:
Newline EOF please

Reply:
also, is the makefile really needed?

Reply:
that's a matter of preference and baseline

baseline - if you have the skills you can do it from memory
preference - it could be part of the doc

my feeling -> developers read code first, then the docs